pysrc/bytewax/serde.py

```diff
@@ -6,6 +6,19 @@
 import jsonpickle
 from typing_extensions import override
 
+try:
+    import jsonpickle.ext.numpy as jsonpickle_numpy
+
+    jsonpickle_numpy.register_handlers()
+except ImportError:
+    pass
```

Comment:
I don't think it's much of a problem to just `pass` here, as any dataflow needing this will be probably importing numpy elsewhere, but just to avoid completely hiding the error, I'd add an INFO or DEBUG log here stating that numpy/pandas support is not installed.